### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JoeOster**

### CODE REVIEW

**Comment by anjgola** [comment]
Can we put this as third-party-programs.txt file instead of README. We need to put other landing page information in README.

File: common/dpc_common.hpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//==============================================================
```
Comment (mkitez) [inline common/dpc_common.hpp:1]: Why are we committing this file here? Isn't it supposed to be inside oneAPI package only?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# oneAPI-samples
+oneAPI-samples
```
Comment (mkitez) [inline README.md:1]: Looks like you are changing header to a paragraph, is this intentional?

File: common/dpc_common.hpp (hunk 1 above)
Comment (JoeOster) [reply common/dpc_common.hpp:1]: per request by Swapna, it is for reference only.

File: README.md (hunk 2 above)
Comment (JoeOster) [reply README.md:1]: This is just a place holder, we are working on a readme, so do not approve the merge as of yet

**Pull request merged**